### PR TITLE
Issue 18255 keep header in form detail page

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        component = fixture.debugElement.componentInstance;
+        routerService = fixture.debugElement.injector.get(DotRouterService);
+        component.app = appData;
+        fixture.detectChanges();
+    });
+
+    it('should set messages/values in DOM correctly', () => {
+        fixture.whenStable().then(() => {
+            expect(
+                fixture.debugElement.query(By.css('.dot-apps-configuration__service-name'))
```
Comment (fmontes) [inline src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.spec.ts:66]:
```
let de: DebugElements =fixture.debugElement;
de.query....
```

File: src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            });
+    }
+
+    /**
+     * Redirects to app configuration listing page
+     *
+     * @param string key
+     * @memberof DotAppsConfigurationDetailComponent
+     */
+    goToApps(key: string): void {
```
Comment (fmontes) [inline src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.ts:45]: This should live in the `this.dotRouterService.goToApps`... I think.

File: src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        {{ app.name }}
+    </span>
+    <div class="dot-apps-configuration__service-key">
+        {{ messagesKey['apps.key'] }}
+        <dot-copy-button [copy]="app.key" [label]="app.key"></dot-copy-button>
+    </div>
+    <span class="dot-apps-configuration__configurations"
+        >{{
+            app.configurationsCount
+                ? app.configurationsCount + ' ' + messagesKey['apps.configurations']
```
[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.html:18]: strings literal are not supported in html

File: src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    >
+        <ngx-md>{{ app.description }}</ngx-md>
+        <a
+            class="dot-apps-configuration__description__link_show-more"
+            *ngIf="app.description.length > 270"
+            (click)="showMore = !showMore"
+            >{{
+                showMore
+                    ? messagesKey['apps.confirmation.description.show.less']
+                    : messagesKey['apps.confirmation.description.show.more']
```
[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.component.html:36]: https://github.com/dotCMS/core-web/pull/1305#discussion_r417512888

**Comment by alfredo-dotcms** [comment]
@fmontes feedback uploaded!

**Pull request merged**